nucypher/datastore/keypairs.py

```diff
@@ -190,7 +190,7 @@ def __init__(self,
         self.certificate_filepath = certificate_filepath
 
     def get_deployer(self, rest_app, port):
-        return HendrixDeployTLS("start",
+        return DeployTLSWithStatics("start",
```

Comment:
What made you decide to create a new object, instead of a feature switch kwarg on `HendrixDeployTLS`?

Reply:
I don't remember but I did this about a year ago and probably wasn't as confident in making large changes to Nucypher or Hendrix at the time...